src/EditorFeatures/Core.Wpf/IWpfTextViewExtensions.cs

```diff
@@ -19,6 +19,12 @@ void firstLayout(object sender, TextViewLayoutChangedEventArgs args)
             {
                 view.VisualElement.Dispatcher.BeginInvoke(new Action(() =>
                 {
+                    var newHeight = view.LineHeight * view.TextBuffer.CurrentSnapshot.LineCount;
+                    var currentHeight = view.VisualElement.Height;
+
+                    if (IsNormal(currentHeight) && newHeight > currentHeight)
```

Comment:
why do you only do this if 'currentHeight' is 'normal'?

Reply:
Mistake, fixed in the last commit.